omod/src/main/webapp/fragments/dashboardwidgets/obsByEncounter.gsp

```diff
@@ -0,0 +1,61 @@
+<style>
+    #obsTable {
+        table-layout: auto;
+        display: block;
+        overflow-x: auto;
+    }
+
+    ::-webkit-scrollbar {
+        -webkit-appearance: none;
+        height: 5px;
+    }
+
+    ::-webkit-scrollbar-thumb {
+        border-radius: 4px;
+        background-color: rgba(0, 0, 0, .5);
+        -webkit-box-shadow: 0 0 1px rgba(255, 255, 255, .5);
+    }
+
+</style>
+
+<div class="info-section encounters-by-type-widget">
+    <div class="info-header">
+        <i class="${app.icon}"></i>
+        <h3>${ ui.message(app.label) }</h3>
+        <a href="${ ui.urlBind("/" + contextPath + app.url, [ "patient.uuid": patient.id ]) }" class="right">
+            <i class="icon-share-alt edit-action" title="Edit"></i>
+        </a>
+    </div>
+    <% if ( encounterObs !=null && (encounterObs.size() > 0 ))  { %>
+        <div class="info-body">
+            <table id="obsTable">
+                <thead>
+                <tr>
+                    <th>${ ui.message("coreapps.date") } </th>
+                    <th>${ ui.message("pihcore.riskFactors") }</th>
```

Comment:
@cioan - this is not correct anymore, now that this is generic.  This needs to probably be something passed in via config.  Can you fix this?

Reply:
Nice catch @mseaton , I missed this one...